tests/scalers/nats-jetstream-helpers.ts

```diff
@@ -0,0 +1,268 @@
+import * as tmp from 'tmp';
```

Comment:
We are finishing migration of e2e tests to Go, could you please rewrite this?

https://github.com/kedacore/keda/issues/2737